### PR TITLE
Rogue & Gambit German Translation

### PR DESCRIPTION
**Pull request by Daedalica**

### CODE REVIEW

**Comment by Daedalica** [comment]
@Kamalisk Ready to merge

File: translations/de/pack/gambit.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
+[
+   {
+      "code": "37001a",
+      "name": "Gambit",
+      "traits": ". X-Men.",
```
Comment (jordanweiler) [inline translations/de/pack/gambit.json:5]: Should this be `Dieb. X-Men.`?

File: translations/de/pack/rogue_encounter.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+   },
+   {
+      "code": "38032",
+      "name": "Wade Cole",
+      "traits": "Reaver.",
+      "text": "Teamwork([[REAVER]])."
+   },
+   {
+      "code": "38033",
+      "name": "Meurray Reese",
```
Comment (jordanweiler) [inline translations/de/pack/rogue_encounter.json:51]: The English version of this card is `Murray Reese` so is `Meurray` a typo?

File: translations/de/pack/gambit.json [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      "text": "<i>Karte aufladen</i> - <b>Aktion</b>: Platziere 1 Ladungsmarker auf Gambit. (Nur ein Mal pro Runde.)\n<i>Karte werfen</i> - <b>Unterbrechung</b>: Sobald du ein [[ANGRIFF]] Ereignis spielst, entferne bis zu 3 Ladungsmarker von Gambit → das Ereignis fügt für jeden entfernten Marker +1 Schaden zu."
+   },
+   {
+      "code": "37001b",
+      "name": "Remy Lebeau",
```
Comment (jordanweiler) [inline translations/de/pack/gambit.json:10]: Should this be `Remy LeBeau`?

File: translations/de/pack/gambit.json [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      "traits": "Dieb. Mutant.",
+      "text": "<i>Meisterdieb</i> - <b>Aktion</b> <i>(Widerstand)</i>: Erschöpfe Remy LeBeau und sieh dir die obersten 2 Karten des Begegnungsdecks an. Lege 1 der Karten ab → entferne Bedrohung von einem Plan in Höhe der Boost-Symbole ([boost]) auf der Karte."
+   },
+   {
+      "code": "37002",
+      "flavor": "\"Ich kann diesen verrückten Cajun nirgendwohin mitnehmen!\"",
+      "name": "Rogue",
+      "subname": "Anna Maria",
```
Comment (jordanweiler) [inline translations/de/pack/gambit.json:18]: Should this be `Anna Marie`?

**Review by jordanweiler — COMMENTED** [review]
I can't pretend that I understand German but I might have found a few typos with character names

**Comment by Daedalica** [comment]
@jordanweiler You are right. I changed it.

**Pull request merged**